### PR TITLE
Lab 11

### PR DESCRIPTION
**Pull request by asbudhkar**
In this lab we looked at Variational Auto-encoders and how they are different from simple Auto-encoders, the working of VAE based on expectation and variance. Also, we learnt about GANs and their working with the analogy of fake money. 
In coding part we did the VAE notebook  which performed a smooth interpolation between inputs in given steps

Authors Yu Cao, Evgenii Nikitin 
Editor - Aishwarya Budhkar 
Date 04/16/2019

### CODE REVIEW

**Comment by Atcold** [comment]
This should be `labs/11`, otherwise you'll be overwriting others' files.
Also, if you can, convert the bullet point into plain English. Thanks.

**Comment by asbudhkar** [comment]
> This should be `labs/11`, otherwise you'll be overwriting others' files.
> Also, if you can, convert the bullet point into plain English. Thanks.

I have done the changes. Please let me know any other changes required.
Thanks

**Comment by Atcold** [comment]
Thanks!

File: labs/11/practice.tex [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    \label{fig:bubbles}
+\end{figure}
+
+\section{Generative Adversarial Network}
+\subsection{Introduction}
+\begin{figure}
+    \centering
+    \includegraphics[width=0.7\textwidth]{"labs/11/images/VAE and GAN".png}
+    \caption{Architecture of GAN}
+    \label{fig:my_label}
```
Comment (Atcold) [inline labs/11/practice.tex:54]: Use meaningful label.

File: labs/11/practice.tex [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+For discriminator, we go from the original input ($x$) or the crafted input ($\hat{x}$) to a learned loss. $\begin{aligned} D : & \mathbb{R}^{n} \rightarrow(0,1) & x \vee \hat{x} \mapsto \ell \end{aligned}$
+
+\subsection{Model illustration}
+
+\begin{figure}
+    \centering
+    \includegraphics[width=0.7\textwidth]{"labs/11/images/GAN illustration".png}
+    \caption{Visualization of encoding and decoding process}
+    \label{fig:my_label}
```
Comment (Atcold) [inline labs/11/practice.tex:73]: You cannot use the same label multiple times. This creates errors.

Comment (asbudhkar) [reply labs/11/practice.tex:73]: I have changed the label names. Thanks

**Pull request merged**